lib/helpers/UiBaseLocators.ts

```diff
@@ -173,14 +168,24 @@ export class UiBaseLocators {
     await this.whereConstrainValueTxt.clear();
     await this.whereConstrainValueTxt.fill(constrainValue);
     await this.whereConstrainValueTxt.press('Enter');
+    await this.submitBtn.click({force: true});
```

Comment:
We should not include this step in this method because we need to verify whether the code is displayed correctly or not before clicking the submit button